source/Application/Controller/Admin/GenericImportMain.php

```diff
@@ -183,7 +183,7 @@ protected function _getCsvFieldsNames()
     {
         $blCsvContainsHeader = $this->getConfig()->getRequestParameter('blContainsHeader');
         \OxidEsales\Eshop\Core\Registry::getSession()->setVariable('blCsvContainsHeader', $blCsvContainsHeader);
-        $sCsvPath = $this->_getUploadedCsvFilePath();
+        $this->_getUploadedCsvFilePath();
```

Comment:
I'm not sure about this one so I only cleared the variable assignment. The called method seems to execute something relevant which is relevant for the current method.